src/spring/HISTORY.md

```diff
@@ -1,5 +1,9 @@
 Release History
 ===============
+1.7.0
+---
+* Add argument `--deployment-name` in command `az spring app create`.
```

Comment:
Do we need to upgrade the secondary version just for adding a new parameter?

Reply:
Sure. Will change 1.7.0 to 1.6.4.